tasks/eva_2357/populate_ids.py

```diff
@@ -0,0 +1,196 @@
+import hashlib
+import argparse
+import pymongo
+import traceback
+import logging
```

Comment:
Suggest using logger from pyutils logging_cfg module since it comes with timestamps too...